app/deed/search/search.mustache

```diff
@@ -0,0 +1,10 @@
+<form id="search-deed-form" action="/deed/search" method="POST">
+    <div class="form-group">
+        <label class="form-label">Reference number</label>
```

Comment:
In order for a label to be 'clickable' it needs to point to the input id - in this example:

```<label class="form-label" for="deed-ref">Reference number</label>```